document-generator-prosecution/src/main/java/uk/gov/companieshouse/document/generator/prosecution/service/ApiClientService.java

```diff
@@ -0,0 +1,64 @@
+package uk.gov.companieshouse.document.generator.prosecution.service;
+
+import java.util.UUID;
+
+import javax.servlet.http.HttpServletRequest;
+
+import org.springframework.stereotype.Component;
+import org.springframework.web.context.request.RequestContextHolder;
+import org.springframework.web.context.request.ServletRequestAttributes;
+
+import uk.gov.companieshouse.api.InternalApiClient;
+import uk.gov.companieshouse.api.http.ApiKeyHttpClient;
+import uk.gov.companieshouse.api.http.HttpClient;
+import uk.gov.companieshouse.environment.EnvironmentReader;
+import uk.gov.companieshouse.environment.impl.EnvironmentReaderImpl;
+
+@Component
+public class ApiClientService {
+
+    private static final EnvironmentReader READER = new EnvironmentReaderImpl();
+
+    private static final String chsApiKey = READER.getMandatoryString("CHS_API_KEY");
+    private static final String apiUrl = READER.getMandatoryString("API_URL");
+    private static final String X_REQUEST_ID_HEADER = "x-request-id";
+
+    public InternalApiClient getApiClient() {
```

Comment:
Why isn't the api-sdk-manager-java-library being used to get an instance of the SDK instead of duplicating the code?

Reply:
Done